content/en/contribute/medic/onboarding/team-meetings.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Team Meetings"
+linkTitle: "Team Meetings"
+weight: 4
+description: >
+  How the Product full-team meetings run
+---
+
+## Why
+Getting to see each other all in one place at the same time, even if only online, lets people get to be around each other, learn from each other, and hear other perspectives. We're real people! ...terrific people too. This is a small space to spend some time around each other as a group. We're mostly trying to learn more about each other, celebrate recent successes, and cover the occasional big news or team-wide changes. It's mostly about being together to smile a bit.  
+
+## When
+Roughly the first Tuesday (or Wednesday, for New Zealand teammates) week of each month
+
+## Who is there
+The whole Product Team.
+
+## Who facilitates it
+Maybe you! We'll have a rotation of teammates leading the call. It doesn't mean you give a 2-hour speech at your teammates. It just means you help out with some basic logistics and guide us through the loose segments so it's not chaos. 
+
+## How to facilitate it
+### Preparations
+1. About 2 weeks before the call, open the [Product Team Call](https://docs.google.com/document/d/14AuJ7SerLuOPESBjQlJqpBtzwSAoVf5ykTT7fjyJBT0/edit#heading=h.tpmdaeagi9ap) document and:
```

Comment:
The document currently needs content. Do you think the facilitator should inform every person responsible for adding their updates?

Reply:
The people would probably be Gareth and me, then whoever is doing the question and the activity person. The latter two are covered, so it's just Gareth and me. I'll update the steps a bit to have that notification covered in the team-wide msg about the jamboard.